server/bin/pbench-state-manager.py

```diff
@@ -108,6 +111,9 @@ def main(options):
         dest="path",
         help="Specify a tarball filename (from which controller and name will be derived)",
     )
+    parser.add_argument(
+        "--user", dest="user", help="Specify the owning username for the dataset",
```

Comment:
Yes, I've allowed myself to be overly influenced by the backpressure of our CLI unit test system. I might as well just drop the concept of unowned datasets.

In fact, now that Nikhil's changes are actually in, I really should _validate_ the user parameter against the Users table instead of just taking it blindly. Ha; I hadn't even considered that until now.